plugins/hls-semantic-tokens-plugin/test/testdata/T1.expected

```diff
@@ -0,0 +1,81 @@
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 6, _startChar = 18, _len = 3}, _name = "Set"}
```

Comment:
I'm not totally sure why this is so much more useful than printing the `SemanticTokenAbsolute`s? We'd see the "final output" rather than the haskell token types you assign, but maybe that's fine?

Reply:
Ah, you include the text under the span! That does seem useful!